### DISCUSSION TITLE
Account not Found

### DISCUSSION THREAD

**Discussion by aarrohan** [vercel/community#2354] [Help] [answered]
### Summary

Hello,

I hope you guys are doing well. I have a vercel account connected with github. But few days ago it suddenly stopped working I mean when I try to login using github it says account not found. I don't have any email connected with that account so now how can I get back the account?

Thanks

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with login, @aarrohan. I have not already done it, please email [registration@vercel.com](mailto:registration@vercel.com). To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account, such as your GitHub username, and the project you are deploying can help accelerate the process.

**Reply by aarrohan** [reply]
I have sent an email at [registration@vercel.com](mailto:registration@vercel.com) but I was told to post the issue on github community.
